package.json

```diff
@@ -14,30 +14,30 @@
     },
     "license": "Apache-2.0",
     "scripts": {
-        "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
-        "install:playground": "cd src/playground && npm install --no-package-lock",
-        "watch:sass": "sass --watch src/assets/scss:src/assets/css",
-        "watch:eleventy": "cross-env IS_DEV=true eleventy --serve --port=2022",
-        "watch:webpack": "webpack watch --mode=development",
+        "build": "npm-run-all install:playground build:sass build:website build:playground build:minify-images",
+        "build:minify-images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
```

Comment:
Will confirm once with https://github.com/eslint/eslint/pull/17431#discussion_r1280829478 this thread and then make changes accordingly.

Reply:
Looks good 👍🏻